app/assets/javascripts/auth/registrations.js

```diff
@@ -0,0 +1,57 @@
+/* global layout_resizer */
+
+$(document).ready(function () {
+  $('#add_application_token_btn').on('click', function (_event) {
```

Comment:
This function can be moved to the function defined in line 21, since both are executed when document is ready.
reference: http://api.jquery.com/jquery/#jQuery3